index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" 
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" 
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <title>WebTekhy | Homepage</title>
+</head>
+<body id="hideOverflow">
+    <header class="header" id="header">
+        <div class="logo">
+            <a href="index.html"><img class="logo-img" src="images/webtekhy-icon.png" alt="webtekhy icon"></a>
+        </div>
+        <nav id="menu_bar" class="navbars">
+            <i id="menuBarImg" class="fa fa-bars" aria-hidden="true"></i>
+            <div id="close_bar" class="close-menu">
+                <i id="close_menu" class="close" >&times;</i>  
+            </div>
+            <ul id="menu_list" class="navis">
+                <li class="nav-item"><a href="about.html" class="nav-link abt-nav">About</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                <li class="nav-item"><a href="#" class="nav-link wt-nav">WT Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="title-section">
+            <div class="details">
+                <p class="quote-text">
+                "Hello! Sharing World"
+                </p>
+                <h1 class="title">
+                    WEBTEKHY UNIVERSAL BOOTCAMP 2022
+                </h1>
+            </div>
+            
+            <div class="descriptions">
+                <p class="description">
+                    A fun festival where people from more than 80 countries who believe in the value of sharing and openness and 
+                    create positive change gather and share their thoughts will be held in Korea in October.
+                </p>
+            </div>
+
+            <h2 class="date">
+                2022.05.16(MON) ~ 20(FRI)
+            </h2>
+            <p class="location">
+                @ National Theartre of Nigeria, Kwara Hotel and more
+            </p>
+        </section>
```

Comment:
- [ ] For the Desktop version both the Program and Title sections seem to have way too little padding to the left and right compared to the Design template as you can see below
> Yours
![image](https://user-images.githubusercontent.com/90556221/169626658-a446112e-4633-4038-bc67-d90471b04c7a.png)

> Design
![image](https://user-images.githubusercontent.com/90556221/169626691-cf15e665-01e5-4bd6-bd68-f57db80aaf75.png)

Kindly ensure you follow the Design template to a fault when it comes to layout.